### PR TITLE
Donna python

### PR DESCRIPTION
**Pull request by donnaaboise**
Add minor changes to check gauges, and so on

### CODE REVIEW

File: src/python/visclaw/plotpages.py [hunk 1]
```diff
@@ -191,7 +191,7 @@ def plots2html(plot_pages_data):
           <title>EagleClaw Plot Index</title>
           <head>
           <link type="text/css" rel="stylesheet"
-                href="http://localhost:50005/eagleclaw/eagleclaw.css">
+                href="http:g//localhost:50005/eagleclaw/eagleclaw.css">
```
Comment (mandli) [inline src/python/visclaw/plotpages.py]: Probably don't want this g here.

File: src/python/visclaw/plotpages.py [hunk 2]
```diff
@@ -717,7 +724,7 @@ def plotclaw2kml(plotdata):
         KML.open(1))
 
     # set all other figures to off.
-    fig_vis = 1
+    # fig_vis = 1
```
Comment (mandli) [inline src/python/visclaw/plotpages.py]: Should this be commented out?

File: src/python/visclaw/plotpages.py [hunk 3]
```diff
@@ -1439,10 +1460,14 @@ def plotclaw2kml(plotdata):
 
     # Create individual level files in subdirectories
 
-    doc_frames = [[0 for j in range(numframes)] for i in range(maxlevels)]
+    try:
+        doc_frames = [[0 for j in range(numframes)] for i in range(0,maxlevels+1-level_base)]
+    except:
+        import pdb
```
Comment (mandli) [inline src/python/visclaw/plotpages.py]: Remove `pdb`.

File: src/python/visclaw/plotpages.py [hunk 4]
```diff
@@ -970,9 +982,12 @@ def plotclaw2kml(plotdata):
 
     try:
         setgauges = gaugetools.read_setgauges(plotdata.outdir)
+        have_gauges = True
     except:
         print("     File gauges.data not found.")
-    else:
+        have_gauges = False
```
Comment (mandli) [inline src/python/visclaw/plotpages.py]: Don't you want to try to read the data and then check the file to see if you have gauges or not?  If you cannot read the gauge.data file at all you cannot do anything below regardless.

**Comment by donnaaboise** [comment]
Hopefully, I addressed these changes.

**Pull request merged**